app/views/components/docs/email-signup-section.yml

```diff
@@ -0,0 +1,34 @@
+name: Email signup section
+description: Renders a box with a link to sign up for email notifications
+shared_accessibility_criteria:
```

Comment:
Have you thought about other accessibility criteria? I'm thinking about the contrast of the text against the background(s) and how the icon should appear/not appear to screen readers.

Reply:
Updated with the criteria you suggested + heading level criteria